### ISSUE TITLE
Unreceived transaction

### ISSUE THREAD

**Issue by Chessies** [kucoin-community-chain/any-advice-issue#22]
I sent USDC to Coinbase from KuCoin using KCC. KCC confirms Tx completed, Coinbase does not support krc-20. The info in KCC says to have it “refund them to your original address” which is mine. How do I do that? Thank you

**Comment by 1218016476** [comment]
I have the same exact problem. did you fix yours?

**Comment by here123actual** [comment]
I have the same problem now. Did you guys solve it, getting the money back? 
Thank you in advance.

**Comment by Chessies** [comment]
No, not fixed yet. Coinbase says there’s nothing they can do as they don’t support kcc transactions, but they might someday in which case the money
Would just “appear”.

**Comment by Chessies** [comment]
Coinbase claims they can’t access the funds so it’s stuck.

**Comment by here123actual** [comment]
Is there any other place where we can see or ask to solve it, or just KCC can do it?

**Comment by Chessies** [comment]
I have no idea.
It’s a good question

**Comment by here123actual** [comment]
Thank you for your response.

I think the funds are gone. I don't see a hope that the funds just gonna appear. How can the fund disappear, I don't even  see my transaction on Kucoin, there is nothing recorded.

**Comment by Chessies** [comment]
The funds would appear if Coinbase stated to utilize the kcc network. The wallet address would still be the same, just the network it’s sent on would then be accepted. For KuCoin, you have to go into your history. You’d find it there.

**Comment by here123actual** [comment]
It is not. I have all transactions there, except this one.

**Comment by Chessies** [comment]
Weird. Are you sure? Check your withdrawals

**Comment by Chessies** [comment]
They’re on separate tabs

**Comment by Chessies** [comment]
![305D6E1B-7005-4333-AFF4-975AAE28B78D](https://user-images.githubusercontent.com/89981335/137965649-1e0d275d-bfe8-45e9-ac55-b9403df3d8a5.jpeg)
![8019241C-2E22-4F01-9F51-DC5B2B9BC395](https://user-images.githubusercontent.com/89981335/137965650-e7958c01-cc50-44d4-a32b-0e73a86658c7.jpeg)
![B43894C0-DE03-4D00-AADF-E999A5623873](https://user-images.githubusercontent.com/89981335/137965651-9f853658-c152-477f-bebe-62ce1d7e298d.jpeg)

**Comment by here123actual** [comment]
Yeah, I see it. But I don't see it in my history.

**Comment by here123actual** [comment]
I could find the transaction ID only when I talked with customer support. They provide that, but I can't find it in my transaction history.

**Comment by Chessies** [comment]
Weird

**Comment by here123actual** [comment]
Yes, very weird.

**Comment by here123actual** [comment]
Thank you so much for texting me back. I really appreciate it.

Now I know, at least what's happening, and let's hope the funds appear.

**Comment by ghost** [comment]
Same thing happened to me a few days ago. Can't get help from anybody. I follow the YouTube video to do this. Seems like it went through for them. The kucoin transaction platform is not user-friendly. Then when your funds don't show up where you send them. They won't help you either. I lost over thirty-five million Shiba Inu with this transaction, but never arrived to coinbase. Same as everybody's issue here. But you would think there would be a resolution somewhere. Hope someone will post an update so everyone can get their money.

**Comment by here123actual** [comment]
Coinbase Wallet is saying that; "Our best recommendation would be to import your wallet using the recovery phrase to a trusted private wallet provider that supports this network to retrieve these funds". I don't know how to do this and if this gonna bring my funds back.
Do you guys have an idea?

**Comment by Chessies** [comment]
No idea how to do that. Let me ask some tech savvy friends.

**Comment by Chessies** [comment]
That’s also Coinbase wallet not Coinbase itself so I don’t know if that will help me. Coinbase wallet is  different from Coinbase

**Comment by here123actual** [comment]
Yes, because Coinbase Wallet accepts Shib coin. Coinbase does not.

**Comment by ghost** [comment]
I'm sorry, but I did not understand a bit of that.

Sent from Yahoo Mail on Android 
 
  On Tue, Oct 26, 2021 at 9:17 AM, ***@***.***> wrote:   


Yes, because Coinbase Wallet accepts Shib coin. Coinbase does not.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Chessies** [comment]
Coinbase and Coinbase wallet handle things differently and have different coins allowed.
